indico/modules/events/editing/schemas.py

```diff
@@ -42,10 +42,14 @@ class EditingFileTypeSchema(mm.ModelSchema):
     class Meta:
         model = EditingFileType
         fields = ('id', 'name', 'extensions', 'allow_multiple_files', 'required', 'publishable', 'is_used',
-                  'filename_template', 'is_used_in_condition')
+                  'filename_template', 'is_used_in_condition', 'url')
 
     is_used = fields.Function(lambda ft: EditingRevisionFile.query.with_parent(ft).has_rows())
     is_used_in_condition = fields.Function(lambda ft: EditingReviewCondition.query.with_parent(ft).has_rows())
+    url = fields.Function(lambda ftype: url_for('.api_edit_file_type',
```

Comment:
I'd keep using `ft` here or change it in the two lines above as well for consistency